besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1180,6 +1180,7 @@ public void run() {
 
     try {
       configureLogging(true);
+      instantiateSignatureAlgorithmFactory();
```

Comment:
I moved the instantiation of the signature algorithm here, to execute it before the native libraries are configured. Otherwise the wrong signature algorithm could be configured in a future release when more than one is supported in Besu.